tests/test_datagen.py

```diff
@@ -0,0 +1,30 @@
+import nndt.space2 as spc
+from nndt.datagen import DataGenForSegmentation
+from tests.base import BaseTestCase, PATH_TEST_ACDC
+from jax.random import PRNGKey, split
+import jax.numpy as jnp
+
+
+class test_datagen(BaseTestCase):
```

Comment:
ThisIsNotACamelStyleClassNotation!

Reply:
Test framework also can work with postfix `TestCase`. Missing this causes a trouble for other developers